libs/ui/elements/src/lib/downloads-list/downloads-list.component.ts

```diff
@@ -33,12 +33,37 @@ export class DownloadsListComponent {
 
   activeFilterFormats: FilterFormat[] = ['all']
 
+  private filterByProtocol(
+    links: DatasetOnlineResource[]
+  ): DatasetOnlineResource[] {
+    const preferredLinks = new Map<string, DatasetOnlineResource>()
+    const OGC_FEATURES_PROTOCOL = 'ogcFeatures'
+
+    links.forEach((link) => {
+      const format = getFileFormat(link)
+      if (!preferredLinks.has(format)) {
+        preferredLinks.set(format, link)
+      } else {
+        const existingLink = preferredLinks.get(format)
+        if (
+          link.accessServiceProtocol === OGC_FEATURES_PROTOCOL &&
+          existingLink?.accessServiceProtocol !== OGC_FEATURES_PROTOCOL
+        ) {
+          preferredLinks.set(format, link)
```

Comment:
this logic will override a download link even if the `name` of the link is different; this might break datasets which have multiple layers offering the same format